pipelines/apDetectorVisitFakesCore.yaml

```diff
@@ -86,4 +86,3 @@ tasks:
       python: |
         from lsst.analysis.tools.atools import FractionFoundFakesDiaSnrMetric
         from lsst.analysis.tools.atools import FractionFoundFakesDiaMagMetric
-
```

Comment:
Please squash this sort of "fixup" commit into "Added fakes metrics"; [commits should be self-contained units](https://developer.lsst.io/work/flow.html#appendix-commit-organization-best-practices).